docs/integrations/optuna.rst

```diff
@@ -3,150 +3,216 @@
-.. note:: You can pass the experiment object as the first argument if you want to do it explicitly.
+    neptune.init(api_token='ANONYMOUS', project_qualified_name='shared/showroom')
+
+.. seealso::
+
+    Instead of logging data to the public project 'shared/onboarding' as an anonymous user 'neptuner' you can log it to your own project.
+    
+    #. Create an account with |Neptune|
+    #. Get your Neptune API token
+    
+       .. image:: ../../_static/images/getting-started/quick-starts/get_token.gif
+            :target: ../../_static/images/getting-started/quick-starts/get_token.gif
+            :alt: Get API token
+          
+    #. Pass the token to ``api_token`` argument of ``neptune.init()``
+    #. Pass your username to the ``project_qualified_name`` argument of the ``neptune.init()`` method: ``project_qualified_name='YOUR_USERNAME/optuna-sweep``.
+    
+       .. code:: python
+          
+          neptune.init(project_qualified_name='YOUR_USERNAME/optuna-sweep', api_token='YOUR_API_TOKEN',)    
+
+Step 2: Create an Experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below to create a Neptune experiment:
 
 .. code-block:: python3
 
-    import neptunecontrib.monitoring.optuna as opt_utils
-
-    neptune_callback = opt_utils.NeptuneCallback()
-
-Pass **neptune_monitor** to **study.optimize**
-----------------------------------------------
-Monitor the metrics and parameters checked at each run:
-
-.. code-block:: python3
+    neptune.create_experiment('optuna-sweep')
 
-    study = optuna.create_study(direction='maximize')
-    study.optimize(objective, n_trials=100, callbacks=[neptune_callback])
+Open the link in a new tab. The charts will currently be empty, but keep the window open. You will be able to see live metrics once logging starts.
 
-Log all results
----------------
-You can log additional information from optuna study after the sweep has completed.
-By running:
+Step 3: Create the Neptune Callback
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. note:: You can pass the experiment object as the first argument if you want to do it explicitly.
 
 .. code-block:: python3
 
-    opt_utils.log_study_info(study)
+   import neptunecontrib.monitoring.optuna as opt_utils
 
-You log the following things to Neptune:
+   neptune_callback = opt_utils.NeptuneCallback()
 
-* Best score
-* Best parameters
-* Interactive plotly figures from optuna.visualization: plot_contour, plot_slice, plot_parallel_coordinate, optimization_history
-* Pickled study object
-
-.. image:: ../_static/images/integrations/optuna_charts.gif
-   :target: ../_static/images/integrations/optuna_charts.gif
-   :alt: Optuna charts in Neptune
-
-Log study and charts after every iteration
-------------------------------------------
-You can log interactive charts from optuna.visualization and the study object after every iteration.
-Simply specify what you want to log at `NeptuneCallback` creation.
+Step 4: Pass the Neptune Callback as a callback to ``study.optimize``
```

Comment:
I'd drop as callback from the header. 
You could explain where to pass it in the text below the header.